uk_bin_collection/tests/test_common_functions.py

```diff
@@ -227,10 +227,10 @@ def mock_os_path_exists(path):
 
 
 def test_create_webdriver_local():
-    result = create_webdriver(None)
+    result = create_webdriver(None, False)
```

Comment:
@robbrad this is a required parameter, which can be seen by the output here: https://github.com/robbrad/UKBinCollectionData/actions/runs/8999755943/job/24722574223?pr=701

Please check that `False` is correct - it appears to be, based on the assumptions in the `create_webdriver` code.